evaluation.md

```diff
@@ -1,2 +1,16 @@
 # Evaluation
 
+##Running Commentary
+We decided that we wanted to keep our language idea from last week. All the reasons that we wanted to create that language were still true, and this provided us with an opportunity to realize the language that we had initially conceived. We started off feeling pretty overwhelmed, as always. The key was writing out a grammar ourselves in BNF to show us where we needed to start from and where we needed to go and how we were gonna get there. Essentially the BNF became the parsing functions.
+
+We pretty much were able to code straight through without any problems (that took >5 minutes) until we got to testing. At this point we ran into a couple of of problems. One, when our parser was reading lines, if the end state was left off then it would read the next start state as the previous rule's end state. We tried to use a new line character to be a line ender but because of the parser's automatic whitespace trimming that made it impossible. We decided that semicolons were our best option.
```

Comment:
No less readable, but it requires the user to 'do more' (in some sense at least). I suppose it really depends on what "more" is to you: with the current version, users can have multiple rules per line at the expense of following each rule with a semicolon; but in our ideal version, we would probably require users to have one rule per line so that they don't have to use semicolons. I'd still tend toward removing semicolons because in most cases it's probably clearer to only have one rule per line.